<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>885</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgba(248, 248, 248, 248);</string>
   </property>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>180</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>音乐分类</string>
    </property>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 流行音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 放松音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 粤语音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 怀旧音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 游戏音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_6">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>330</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 国外音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s2.png</normaloff>:/image/s2.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>180</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>我的音乐</string>
    </property>
    <widget class="QPushButton" name="pushButton_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string> 所有音乐</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s1.png</normaloff>:/image/s1.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>520</y>
      <width>180</width>
      <height>331</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>我的歌单</string>
    </property>
    <widget class="QPushButton" name="pushButton_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  歌单1</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s3.png</normaloff>:/image/s3.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>90</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  歌单2</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s3.png</normaloff>:/image/s3.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  歌单3</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s3.png</normaloff>:/image/s3.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  歌单4</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s3.png</normaloff>:/image/s3.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>180</width>
       <height>60</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>  歌单5</string>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/s3.png</normaloff>:/image/s3.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>1020</width>
      <height>680</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="currentIndex">
     <number>11</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QListWidget" name="listWidget_13">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1020</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_4">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_14">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_15">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_6">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_16">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_7">
     <widget class="QListWidget" name="listWidget_17">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_8">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <widget class="QListWidget" name="listWidget_18">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_9">
     <widget class="QListWidget" name="listWidget_19">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_10">
     <widget class="QListWidget" name="listWidget_20">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_11">
     <widget class="QListWidget" name="listWidget_21">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_12">
     <widget class="QListWidget" name="listWidget_22">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_13">
     <widget class="QListWidget" name="listWidget_23">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_14">
     <widget class="QListWidget" name="listWidget_24">
      <property name="geometry">
       <rect>
        <x>-1</x>
        <y>0</y>
        <width>1022</width>
        <height>680</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>790</y>
      <width>1020</width>
      <height>80</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>1020</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="pushButton_25">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>15</y>
       <width>25</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/left.png</normaloff>:/image/left.png</iconset>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_26">
     <property name="geometry">
      <rect>
       <x>65</x>
       <y>15</y>
       <width>25</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/right.png</normaloff>:/image/right.png</iconset>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>15</y>
       <width>211</width>
       <height>30</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="whatsThis">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="inputMask">
      <string/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_27">
     <property name="geometry">
      <rect>
       <x>850</x>
       <y>15</y>
       <width>31</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="res.qrc">
       <normaloff>:/image/search.png</normaloff>:/image/search.png</iconset>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_155">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>歌曲名称</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_161">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>70</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>歌手</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_160">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>70</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>专辑</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_156">
    <property name="geometry">
     <rect>
      <x>855</x>
      <y>70</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>时长</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_162">
    <property name="geometry">
     <rect>
      <x>1090</x>
      <y>70</y>
      <width>41</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>操作</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <widget class="QMenu" name="menu_5">
     <property name="title">
      <string>导入音乐</string>
     </property>
     <addaction name="actiona_5"/>
     <addaction name="actionb_2"/>
    </widget>
    <widget class="QMenu" name="menu_6">
     <property name="title">
      <string>导出音乐</string>
     </property>
     <addaction name="actiona_6"/>
     <addaction name="actionb_3"/>
    </widget>
    <addaction name="menu_5"/>
    <addaction name="menu_6"/>
    <addaction name="actionc"/>
    <addaction name="actiond_2"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>工具</string>
    </property>
    <addaction name="actiona_2"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>设置</string>
    </property>
    <addaction name="actiona_3"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actiona_4"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="action2">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="toolTip">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action3">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action4">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action1_2">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action1_3">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="action1_4">
   <property name="text">
    <string>关于作者</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actiond">
   <property name="text">
    <string>导入单首音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actione">
   <property name="text">
    <string>导入文件内所有音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="actionc">
   <property name="text">
    <string>通过网页链接播放音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiond_2">
   <property name="text">
    <string>通过歌单链接播放音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiona_2">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiona_3">
   <property name="text">
    <string>待开发</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiona_4">
   <property name="text">
    <string>关于作者</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiona_5">
   <property name="text">
    <string>导入单首音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionb_2">
   <property name="text">
    <string>导入文件夹内所有音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actiona_6">
   <property name="text">
    <string>导出单首音乐</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
  <action name="actionb_3">
   <property name="text">
    <string>导出所有音乐至文件夹</string>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
